Float32MultiArrayStamped x_0
Float32MultiArrayStamped xb_traj
---
Float32MultiArrayStamped u_traj
Float32MultiArrayStamped x_traj
